tests/sagemaker-tests/test_sagemaker_utils.py

```diff
@@ -0,0 +1,193 @@
+        json_path = os.path.join(tmpdirname, 'test.json')
+
+        create_dummy_file(entry_path)
+        create_dummy_file(model_path)
+        create_dummy_file(yaml_path)
+        create_dummy_file(json_path)
+        output_path = os.path.join(tmpdirname, 'output_folder')
+        create_dummy_file(output_path)        
+        with pytest.raises(AssertionError, match='Output path should be a folder name, but got'):
+            wrap_model_artifacts(model_path, yaml_path, json_path, entry_path,
+                                        output_tarfile_name='model', output_path=output_path)
+    
+
+    #     2.4: given folders instead of files
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        entry_path = os.path.join(tmpdirname, 'nc_infer_entry.py')
+        model_path = os.path.join(tmpdirname, 'model.bin')
+        yaml_path = os.path.join(tmpdirname, 'test.yaml')
+        json_path = os.path.join(tmpdirname, 'test.json')
+
+        # create a entry point file as a folder
+        os.makedirs(entry_path)
+        create_dummy_file(model_path)
+        create_dummy_file(yaml_path)
+        create_dummy_file(json_path)
+        with pytest.raises(AssertionError, match='SageMaker entry point file, .* got a folder'):
+            wrap_model_artifacts(model_path, yaml_path, json_path, entry_path,
+                                        output_tarfile_name='model')
+        os.rmdir(entry_path)
+
+        create_dummy_file(entry_path)
+        # create a model file as a folder
+        os.remove(model_path)
+        os.makedirs(model_path)
+        create_dummy_file(yaml_path)
+        create_dummy_file(json_path)
+        with pytest.raises(AssertionError, match='model file, .* got a folder'):
+            wrap_model_artifacts(model_path, yaml_path, json_path, entry_path,
+                                        output_tarfile_name='model')
+        os.rmdir(model_path)
+
+        create_dummy_file(entry_path)
+        create_dummy_file(model_path)
+        # create a yaml file as a folder
+        os.remove(yaml_path)
+        os.makedirs(yaml_path)
+        create_dummy_file(json_path)
+        with pytest.raises(AssertionError, match='model configuration YAML file, .* got a folder'):
+            wrap_model_artifacts(model_path, yaml_path, json_path, entry_path,
+                                        output_tarfile_name='model')
+        os.rmdir(yaml_path)
+
+        create_dummy_file(entry_path)
+        create_dummy_file(model_path)
+        create_dummy_file(yaml_path)
+        # create a json file as a folder
+        os.remove(json_path)
+        os.makedirs(json_path)
+        with pytest.raises(AssertionError, match='graph metadata JSON file, .* got a folder'):
+            wrap_model_artifacts(model_path, yaml_path, json_path, entry_path,
+                                        output_tarfile_name='model')
+
+if __name__ == '__main__':
+    test_wrap_model_artifacts()
```

Comment:
We use `pytest` to run tests so this is not needed

Reply:
removed.